src/tokens/TokenBalanceItem.test.tsx

```diff
@@ -0,0 +1,70 @@
+import { render } from '@testing-library/react-native'
+import BigNumber from 'bignumber.js'
+import React from 'react'
+import { Provider } from 'react-redux'
+import { TokenBalance } from 'src/tokens/slice'
+import { TokenBalanceItem } from 'src/tokens/TokenBalanceItem'
+import { createMockStore } from 'test/utils'
+import { mockCusdAddress } from 'test/values'
+
+let mockTokenInfo: TokenBalance
+
+describe('TokenBalanceItem', () => {
```

Comment:
can we add a test for when the local currency amount is `--`?

Reply:
Added in https://github.com/valora-inc/wallet/pull/4207/commits/b0f64cda6fc9512fd4580084c08f6cb74bfc925e!